packages/legend-query-builder/src/components/shared/CustomDatePicker.tsx

```diff
@@ -840,22 +840,21 @@ const AbsoluteTimeValueSpecificationEditor: React.FC<{
   const updateAbsoluteTimeValue: React.ChangeEventHandler<HTMLInputElement> = (
     event,
   ) => {
+    //
+    const value = new Date(event.target.value).getUTCSeconds()
```

Comment:
wont this always return 0, even they have specified the seconds ? 
`new Date(event.target.value).getUTCSeconds() `